src/middlewares/authenticateUser.js

```diff
@@ -0,0 +1,29 @@
+const JWT = require("jsonwebtoken");
+const authenticateUser = (req, res, next) => {
+  const { token } = req.cookies;
+
+  // verify if token is present or not
+  if (!token) {
+    return res.status(400).json({
```

Comment:
Typically a "not authorized" response would return a 403 status code. 403 = Forbidden.

Reply:
Yes ! 
Thanks you sir , i will take it !